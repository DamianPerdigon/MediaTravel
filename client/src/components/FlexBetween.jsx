import { Box } from '@mui/material';
import { styled } from '@mui/system';

// Styles for FlexBetween
const flexBetweenStyles = {
  display: 'flex',
  justifyContent: 'space-between',
  alignItems: 'center',
};

const FlexBetween = styled(Box)(flexBetweenStyles);

export default FlexBetween;
export { FlexBetween };
