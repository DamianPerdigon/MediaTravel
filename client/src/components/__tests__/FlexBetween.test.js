import React from 'react';
import { render } from '@testing-library/react';
import FlexBetween from '../FlexBetween'; // Asegúrate de que la ruta de importación sea correcta

describe('FlexBetween', () => {
  it('debe coincidir con el snapshot', () => {
    const { asFragment } = render(<FlexBetween />);

    expect(asFragment()).toMatchSnapshot();
  });
});
