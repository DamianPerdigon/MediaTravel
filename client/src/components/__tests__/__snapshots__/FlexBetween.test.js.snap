// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexBetween debe coincidir con el snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-1omog99"
  />
</DocumentFragment>
`;
